examples/cursors.py

```diff
@@ -1,104 +1,260 @@
-    s2 = []
-    for line in arrow:
-        s2.append(line.replace("x", "X").replace(",", ".").replace("O", "o"))
-    cursor, mask = pg.cursors.compile(s2, "X", ".", "o")
-    size = len(arrow[0]), len(arrow)
-    pg.mouse.set_cursor(size, hotspot, cursor, mask)
+
+# Calculate if mouse position is inside circle
+def check_circle(mouse_pos_x, mouse_pos_y, center_x, center_y, radius):
+    return (mouse_pos_x - center_x) ** 2 + (mouse_pos_y - center_y) ** 2 < radius**2
 
 
 def main():
+
     pg.init()
+    pg.display.set_caption("Cursors Example")
+
     pg.font.init()
-    font = pg.font.Font(None, 24)
-    bg = pg.display.set_mode((800, 600), 0, 24)
-    bg.fill((255, 255, 255))
-    bg.blit(font.render("Click to advance", 1, (0, 0, 0)), (0, 0))
+    font = pg.font.Font(None, 30)
+    font1 = pg.font.Font(None, 24)
+
+    bg = pg.display.set_mode((500, 400))
+    bg.fill((183, 201, 226))
+
+    # Initialize circles
+    radius1 = 40
+    radius2 = 40
+    radius3 = 40
+    radius4 = 40
+    radius5 = 40
+    radius6 = 40
+    radius7 = 40
+
+    pos_x1 = 82
+    pos_x2 = 138
+    pos_x3 = 194
+    pos_x4 = 250
+    pos_x5 = 306
+    pos_x6 = 362
+    pos_x7 = 418
+
+    pos_y1 = 140
+    pos_y2 = 220
+    pos_y3 = 140
+    pos_y4 = 220
+    pos_y5 = 140
+    pos_y6 = 220
+    pos_y7 = 140
+
+    circle1 = pg.draw.circle(bg, (255, 255, 255), (pos_x1, pos_y1), radius1)
+    circle2 = pg.draw.circle(bg, (255, 255, 255), (pos_x2, pos_y2), radius2)
+    circle3 = pg.draw.circle(bg, (255, 255, 255), (pos_x3, pos_y3), radius3)
+    circle4 = pg.draw.circle(bg, (255, 255, 255), (pos_x4, pos_y4), radius4)
+    circle5 = pg.draw.circle(bg, (255, 255, 255), (pos_x5, pos_y5), radius5)
+    circle6 = pg.draw.circle(bg, (255, 255, 255), (pos_x6, pos_y6), radius6)
+    circle7 = pg.draw.circle(bg, (255, 255, 255), (pos_x7, pos_y7), radius7)
+
+    # Initialize button
+    button_text = font1.render("Click here to change cursor", True, (0, 0, 0))
+    button = pg.draw.rect(
+        bg,
+        (180, 180, 180),
+        (139, 300, button_text.get_width() + 5, button_text.get_height() + 50),
+    )
+    button_text_rect = button_text.get_rect(center=button.center)
+    bg.blit(button_text, button_text_rect)
+
     pg.display.update()
-    for cursor in [no, arrow]:
-        TestCursor(cursor)
-        going = True
-        while going:
-            pg.event.pump()
-            for e in pg.event.get():
-                if e.type == pg.MOUSEBUTTONDOWN:
-                    going = False
-    pg.quit()
+
+    cursors = [
+        system_cursor1,
+        color_cursor,
+        system_cursor2,
+        image_cursor,
+        system_cursor3,
+        bitmap_cursor1,
+        bitmap_cursor2,
+    ]
+
+    index = 0
+    pg.mouse.set_cursor(cursors[index])
+
+    pressed = False
+    clock = pg.time.Clock()
+
+    while True:
+
+        clock.tick(50)
+
+        mouse_pos = pg.mouse.get_pos
+        mouse_x = mouse_pos()[0]
+        mouse_y = mouse_pos()[1]
```

Comment:
```
        mouse_pos = pg.mouse.get_pos
        mouse_x = mouse_pos()[0]
        mouse_y = mouse_pos()[1]
```
this can be written more elegantly as

```
        mouse_x, mouse_y = pg.mouse.get_pos()
```